components/user_settings/general/user_settings_general.jsx

```diff
@@ -297,6 +298,23 @@ class UserSettingsGeneralTab extends React.Component {
         );
     }
 
+    setDefaultProfilePicture = async (e) => {
+        e.preventDefault();
```

Comment:
There's a chance that this might cause React to print a warning if this gets pre-empted because it's an async function and React doesn't like when you hold onto an event. Maybe we could have SettingPicture call `e.preventDefault()` and not pass it back to `this.props.onSetDefault()`?

Reply:
Sure, I can do that.

Reply:
Done! PTAL